examples/rustapi_module/tests/test_datetime.py

```diff
@@ -80,7 +83,8 @@ def test_invalid_date_fails():
         rdt.make_date(2017, 2, 30)
 
 
-@given(d=dates())
+# Feeding this tests dates from too early will cause `get_timestamp` to raise.
+@given(d=st.dates(min_value=pdt.date(1900, 1, 1)))
```

Comment:
If this only happens on PyPy, can you make it conditional on that? Same with below.

Even better would be to make this test `xfail` on PyPy:

```python
# get_timestamp fails on Pypy if the dates are too early
if PYPY and dt < pdt.date(1900, 1, 1):
    pytest.xfail("get_timestamp will raise on PyPy with dates before 1900")
```

This seems like a bug in Pypy, and we should not encode these restrictions into our code for all platforms.

Reply:
👍